docs/getting-started.asciidoc

```diff
@@ -0,0 +1,224 @@
+
+The visualization tools available on the *Visualize* tab enable you to display aspects of your data sets in several 
+different ways. Visualizations depend on Elasticsearch {ref}/search-aggregations.html[aggregations] in two different 
+types: _bucket_ aggregations and _metric_ aggregations. A bucket aggregation sorts your data according to criteria you 
+specify. For example, in our accounts data set, we can establish a range of account balances, then display what 
+proportions of the total fall into which range of balances.
+
+Click on the *Visualize* tab to start:
+
+image::images/tutorial-visualize.png[]
+
+Click on *Pie chart*, then *From a new search*. Select the `ba*` index pattern. The whole pie displays, since we 
+haven't specified any buckets yet.
+
+image::images/tutorial-visualize-pie-1.png[]
+
+Select *Split Slices* from the *Select buckets type* list, then select *Range* from the *Aggregation* drop-down 
+selector. Select the *balance* field from the *Field* drop-down, then click on *Add Range* four times to bring the 
+total number of ranges to six. Enter the following ranges:
+
+[source,text]
+0            1000
+1000         3000
+3000         7000
+7000        15000
+15000       31000
+31000       50000
+
+Click the green *Apply changes* to display the chart:
+
+image::images/tutorial-visualize-pie-2.png[]
+
+This shows you what proportion of the 1000 accounts fall in these balance ranges. To see another dimension of the data, 
+we're going to add another bucket aggregation. We can break down each of the balance ranges further by the account 
+holder's age.
+
+Click *Add sub-buckets* at the bottom, then select the *Terms* aggregation and the *age* field from the drop-downs. 
+Click the green *Apply changes* button to add an external ring with the new results.
+
+image::images/tutorial-visualize-pie-3.png[]
+
+Save this chart by clicking the *Save Visualization* button to the right of the search field. Name the visualization
+_Pie Example_.
+
+Next, we're going to make a bar chart. Click on *New Visualization*, then *Vertical bar chart*. Select *From a new 
+search* and the `ba*` index pattern, just as you did for the pie chart. You'll see a single big bar, since we haven't
+defined any buckets yet:
+
+image::images/tutorial-visualize-bar-1.png[]
+
+For the Y-axis metrics aggregation, select *Average*, with *age* as the field. For the X-Axis buckets, select the 
+*Range* aggregation and define the same ranges as you did for the pie chart.
+
+Now, click *Add sub-buckets* and *Split Bars* to refine our data. In addition to listing the average age of the 
+accounts in each balance range, we're going to split the bars by the top five states with the highest average ages.
+Select *Terms* as the sub-aggregation, with *state* as the field. Leave the other elements at their default values and 
+click the green *Apply changes* button. Your chart should now look like this:
+
+image::images/tutorial-visualize-bar-2.png[]
```

Comment:
This is the problem with "random" data, its so smooth. See how all the blocks are the same size?